synapse/lib/stormhttp.py

```diff
@@ -144,7 +145,14 @@ def __init__(self, valu, path=None):
     def getObjLocals(self):
         return {
             'json': self._httpRespJson,
+            'msgpack': self._httpRespMsgpack,
```

Comment:
This needs a docstring in ``_storm_locals``

Reply:
:+1: